packages/libs/error-stack/src/compat/mod.rs

```diff
@@ -0,0 +1,30 @@
+#[cfg(feature = "anyhow")]
+mod anyhow;
+#[cfg(feature = "eyre")]
+mod eyre;
+
+/// Compatibility trait to convert from external libraries to [`Report`].
+///
+/// *Note*: Is't not possible to implement [`IntoReport`] or [`Context`] on other error libraries
+/// types as both traits have blanked implementation relying on [`Error`]. This means, that they
```

Comment:
```suggestion
/// types as both traits have blanket implementation relying on [`Error`]. This means that they
```

Do you mean blanket? Or perhaps blank?